app/components/job/details/job-details.module.ts

```diff
@@ -0,0 +1,31 @@
+import { NgModule } from "@angular/core";
+
+import { commonModules } from "app/common";
+import { FileBrowseModule } from "app/components/file/browse";
+import { FileDetailsModule } from "app/components/file/details";
+import { TaskBrowseModule } from "app/components/task/browse";
+
+import { JobErrorDisplayComponent } from "./error-display";
+import { JobDetailsHomeComponent } from "./job-details-home.component";
+import { JobDetailsComponent } from "./job-details.component";
+import { JobEnvironmentSettingsComponent } from "./job-env-settings.component";
+import { JobMetadataComponent } from "./job-metadata.component";
+import { JobPropertiesComponent } from "./job-properties.component";
+
+const components = [
```

Comment:
am wondering if we should make this job.module rather than job details only. then each area (job, task, app, ..) can have a single module and that way we easily know where the modules are when adding new bits.